### ISSUE TITLE
Errors installing on Raspbian Stretch Lite (2018-11-13)

### ISSUE THREAD

**Issue by dr3x** [NikolaiT/se-scraper#16]
sudo apt-get install gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget

pi@raven:~ $ **npm install se-scraper**
(node:1569) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
npm ERR! tar.unpack untar error /home/pi/.npm/puppeteer/1.14.0/package.tgz
npm ERR! tar.unpack untar error /home/pi/.npm/ms/2.1.1/package.tgz
npm ERR! error rolling back Error: ENOTEMPTY: directory not empty, rmdir '/home/pi/node_modules/se-scraper/node_modules/chai/lib/chai/interface'
npm ERR! error rolling back  se-scraper@1.2.16 { Error: ENOTEMPTY: directory not empty, rmdir '/home/pi/node_modules/se-scraper/node_modules/chai/lib/chai/interface'
npm ERR! error rolling back   errno: -39,
npm ERR! error rolling back   code: 'ENOTEMPTY',
npm ERR! error rolling back   syscall: 'rmdir',
npm ERR! error rolling back   path: '/home/pi/node_modules/se-scraper/node_modules/chai/lib/chai/interface' }
npm ERR! Error: Method Not Allowed
npm ERR!     at errorResponse (/usr/share/npm/lib/cache/add-named.js:260:10)
npm ERR!     at /usr/share/npm/lib/cache/add-named.js:203:12
npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:167:7)
npm ERR!     at FSReqWrap.oncomplete (fs.js:135:15)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 4.14.98-v7+
npm ERR! command "/usr/bin/node" "/usr/bin/npm" "install" "se-scraper"
npm ERR! cwd /home/pi
npm ERR! node -v v8.11.1
npm ERR! npm -v 1.4.21
npm ERR! code E405
npm ERR! tar.unpack untar error /home/pi/.npm/lodash/4.17.11/package.tgz
npm ERR! tar.unpack untar error /home/pi/.npm/domhandler/2.4.2/package.tgz
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /home/pi/npm-debug.log
npm ERR! not ok code 0
pi@raven:~ $

pi@raven:~ $ node --version **v8.11.1**
pi@raven:~ $ npm --version **1.4.21**

**Comment by dr3x** [comment]
More info - attempted to install puppeteer on the device to see if I could get that working; that did not work with the embedded chromium but I was able to install chromium-browser and then set puppeteer's executable path to that:

`  const browser = await puppeteer.launch({
    executablePath: '/usr/bin/chromium-browser'
  });`

After this the code samples for puppeteer worked.

Also - before I installed puppeteer directly, as I was clearing out the node modules left from the se-scraper install there was an unmet dependency error for se-scraper:
pi@raven:~ $ npm ls --depth=0
(node:4049) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
/home/pi
...
├── UNMET DEPENDENCY se-scraper@*
...
